types/potrace/index.d.ts

```diff
@@ -0,0 +1,100 @@
+// Type definitions for potrace 2.1
+// Project: https://github.com/tooolbox/node-potrace
+// Definitions by: Direnc Timur <https://github.com/proohit>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import 'node';
```

Comment:
```suggestion
/// <reference types="node" />
```
this is established pattern here to reference Node types

Reply:
fixed in  2217278